Exif IPTC/5.8.0/metadata/asset/images/images.dcl

```diff
@@ -0,0 +1,20 @@
+resource item_security metafield_group_images__system {
+    accessor_item_id = data.member.system.item_id
+    item_id = resource.metafield_group.images.item_id
+    read = true
+    write = true
+}
+
+resource metafield_group images {
+    name = 'Images'
+    description = ''
+    show_in_list = false
+    folder_id = resource.meta_group_folder.images.id
+    sort_index = 50035
+    restrict_to_asset = 'Image'
+    iterative = false
+    parent_group_id = data.metafield_group.asset_info.metafield_group_id
+    visibility_regex = ''
+    visibility_metafield_id = 0
+}
```

Comment:
I'm not sure that I think it makes too much sense to have an `Images` group nested inside the `Images` meta group folder. Maybe we could come up with a better name?

That being said, I know that this might be annoying to change, and that we therefore might not want to do it.

Reply:
That is just the existing structure that was exported, and yes, it's crap :D